README.md

```diff
@@ -40,6 +40,18 @@ Then open http://localhost:9000/.
 
 If you have troubles trying to run the console, refer to their doc: https://github.com/openshift/console/#openshift-no-authentication.
 
+### Loki setup
+
+WIP / FIXME
+
+Currently just dev mode is supported to fetch data from Loki. It means that you need to run the commands described above to run the plugin (`make serve` / `make bridge`).
+
+You also need to deploy Loki and port-forward on your host: `oc port-forward service/loki 3100:3100`
+
+(You don't need Grafana)
+
+For next steps, we'll need to use the console proxy feature to fetch in-cluster Loki, which will be supported via this story: https://issues.redhat.com/browse/CONSOLE-2892
```

Comment:
I didn't know such feature, could we have used it to proxy request from the client directly to loki ?
In this case, did we need the go backend ?

Reply:
Yeah, could be.
I'd suggest that we keep the go backend at the moment, and revisit that choice once all the plugin's "mvp" features are done (so until this epic https://issues.redhat.com/browse/NETOBSERV-85 is done). So that if we find areas where some backend logic makes sense, we don't have to once again reintroduce that code. Does it make sense?

Reply:
Yes, it makes sense!